src/BlazorAdmin/Services/CatalogItemService/Create.cs

```diff
@@ -1,7 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net;
+﻿using System.Net;
```

Comment:
Seems odd to me to have CatalogItemService as the folder, not class name for these operations (and for the folder name to be singular not plural).

Reply:
Done in the next PR #430